model/fn-execution/src/main/proto/beam_provision_api.proto

```diff
@@ -54,7 +54,10 @@ message ProvisionInfo {
     string job_id = 1;
     // (required) The job name.
     string job_name = 2;
-    // (required) The worker ID.
+    // (required) The worker ID. Often this will be the hostname.
```

Comment:
Whose hostname? The runner's? What do we expect SDKs to do with this?

I thought that we instead wanted to make this optional and have ProvisionRequests take a worker id.

Reply:
To be clear, I think this is sufficient for now. I just want to make it explicit that the SDK should not treat this as a unique runner id or anything of the sort.